client/src/pages/ParentPortal.js

```diff
@@ -13,29 +18,52 @@ class PaymentPortal extends Component {
     super(props);
     
     this.state = {
-      announcements: this.props.announcements
+      students: []
     }
+
+    this.ReactIsInDevelomentMode = this.ReactIsInDevelomentMode.bind(this);
+    
   }
 
-  getStudentInfo(){
-    //TODO: Make an API call to fetch all students of the households
+  /**
+   * Makes a GET request to either dev or prod and fetches the student list
+   * as soon as the component mounts and updates the state
+   */
+  componentDidMount(){
+    //TODO: Figure out how it is being passed later : this.props.houseHoldID
+    //TODO: Add logic for retrieving householdID
+    if (this.ReactIsInDevelomentMode()){
+      //Fetch the student list
+      let urlToFetch = API_STUDENT_DEV + '5c8680ffad46ec4f26e7b46f';
```

Comment:
Same as comment as in announcements, simplify by having urlToFetch be the only thing being changed. Also what is "5c8680ffad46ec4f26e7b46f"?

Reply:
It should be the household id for a dummy entry. Add a TODO to replace this with household_id as pulled from Redux.